src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -99,4 +136,152 @@ private void copyUrl() {
+        );
+        helpCards.add(exitCommandCard);
+
+        CommandHelpCard findCommandCard = new CommandHelpCard(
+            FindCommand.COMMAND_WORD, FindCommand.MESSAGE_USAGE, "find bob"
+        );
+        helpCards.add(findCommandCard);
+
+        CommandHelpCard helpCommandCard = new CommandHelpCard(
+            HelpCommand.COMMAND_WORD, HelpCommand.MESSAGE_USAGE, "help"
+        );
+        helpCards.add(helpCommandCard);
+
+        CommandHelpCard inspectCommandCard = new CommandHelpCard(
+            InspectCommand.COMMAND_WORD, InspectCommand.MESSAGE_USAGE, "inspect"
+        );
+        helpCards.add(inspectCommandCard);
+
+        CommandHelpCard listCommandCard = new CommandHelpCard(
+            ListCommand.COMMAND_WORD, "Lists all persons", "list"
+        );
+        helpCards.add(listCommandCard);
+
+        return helpCards;
+    }
+
+    /**
+     * Returns help cards for inspect window
+     */
+    private ArrayList<CommandHelpCard> inspectHelpCards() {
+        ArrayList<CommandHelpCard> helpCards = new ArrayList<>();
+
+        CommandHelpCard addCommandCard = new CommandHelpCard(
+            AddCommand.COMMAND_WORD, AddCommand.MESSAGE_USAGE_DELIVERY, AddCommand.ADD_DELIVERY_EXAMPLE
+        );
+        helpCards.add(addCommandCard);
+
+        CommandHelpCard archiveCommandCard = new CommandHelpCard(
+            ArchiveCommand.COMMAND_WORD, ArchiveCommand.MESSAGE_USAGE, "archive 1"
+        );
+        helpCards.add(archiveCommandCard);
+
+        CommandHelpCard deleteCommandCard = new CommandHelpCard(
+            DeleteCommand.COMMAND_WORD, DeleteCommand.MESSAGE_USAGE_DELIVERY, "delete 1"
+        );
+        helpCards.add(deleteCommandCard);
+
+        CommandHelpCard editCommandCard = new CommandHelpCard(
+            EditCommand.COMMAND_WORD, EditCommand.INSPECT_MESSAGE_USAGE, EditCommand.DELIVERY_EXAMPLE
+        );
+        helpCards.add(editCommandCard);
+
+        CommandHelpCard exitCommandCard = new CommandHelpCard(
+            ExitCommand.COMMAND_WORD, "Closes the app", "exit"
+        );
+        helpCards.add(exitCommandCard);
+
+        CommandHelpCard helpCommandCard = new CommandHelpCard(
+            HelpCommand.COMMAND_WORD, HelpCommand.MESSAGE_USAGE, "help"
+        );
+        helpCards.add(helpCommandCard);
+
+        CommandHelpCard listCommandCard = new CommandHelpCard(
+            ListCommand.COMMAND_WORD, "Lists all persons", "list"
+        );
+        helpCards.add(listCommandCard);
+
+        CommandHelpCard sortAscendingCommandCard = new CommandHelpCard(
+            SortCommand.COMMAND_WORD_ASCENDING, SortCommand.MESSAGE_USAGE_ASCENDING, "asort cost"
+        );
+        helpCards.add(sortAscendingCommandCard);
+        CommandHelpCard sortDescendingCommandCard = new CommandHelpCard(
+            SortCommand.COMMAND_WORD_DESCENDING, SortCommand.MESSAGE_USAGE_DESCENDING, "dsort cost"
+        );
+        helpCards.add(sortDescendingCommandCard);
```

Comment:
Should the example "asort cost" and "dsort cost" contain the prefix **by/** in between? 
For example, "asort by/ cost" and "dsort by/ cost".

Reply:
Apologies. Will fix in soon.